hack/app_sre_create_image_catalog.sh

```diff
@@ -9,8 +9,11 @@ _OPERATOR_NAME="route-monitor-operator"
 BRANCH_CHANNEL="$1"
 QUAY_IMAGE="$2"
 
+# Build an image locally that has all tools we need
+docker build -f hack/pipeline.dockerfile -t pipelinebuilder:latest ./hack/
+
 # generate the bundle folder
-make bundle
+make bundle OPERATOR_SDK_COMMAND="docker run --rm -w /root/tmp -i -v $(pwd):/root/tmp pipelinebuilder:latest"
```

Comment:
we would also need to override the binary to kustomize with `KUSTOMIZE=`

Reply:
yes you're right